cluster/gce/container-linux/configure-helper.sh

```diff
@@ -82,6 +84,22 @@ function get-calico-typha-cpu {
   echo "${typha_cpu}"
 }
 
+# Prepare node labels specifically for master or node.
+function prepare-node-labels {
```

Comment:
> move this to config-default.sh and config-test.sh.

Before closing this PR, not sure I understand how to do so...I thought config-default.sh and config-test.sh happen before the we actually know it is on master or node (KUBERNETES_MASTER is not set yet)?